### PR TITLE
Various updates - Installing

### PR DESCRIPTION
**Pull request by Lennonka**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Cherry-picks vary by commit, see commit messages. (I'll do them once approved.)

### CODE REVIEW

File: guides/common/modules/ref_capsule-server-scalability-considerations.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 = {SmartProxyServer} Scalability Considerations
 
 The maximum number of {SmartProxyServer}s that {ProjectServer} can support has no fixed limit.
-The tested limit is 17 {SmartProxyServer}s with 2 vCPUs on a {ProjectServer} with {RHEL} 8.
+The tested limit is 17 {SmartProxyServer}s with 2 vCPUs on a {ProjectServer}.
```
Comment (Lennonka) [inline guides/common/modules/ref_capsule-server-scalability-considerations.adoc:7]: I'm wondering if we should remove this altogether until we re-test it.

**Comment by Lennonka** [comment]
Fololwup to #1748

File: guides/common/modules/ref_capsule-server-scalability-considerations.adoc (hunk 1 above)
Comment (ekohl) [reply guides/common/modules/ref_capsule-server-scalability-considerations.adoc:7]: You could tone it down and say something like "17 proxies w/ 2 CPUs has been tested".

**Review by ekohl — APPROVED** [review]
Content wise OK, though I'm not too happy about the "CP to x.y" in commit messages. Especially that they're in the title. If it was the body, it was better but possibly leave them out altogether.

**Review by maximiliankolb — APPROVED** [review]
As said by Ewoud, maybe you can just shorten the commit messages and instead keep the information for cherry-picking in this PR. You'll only really need it once immediately after merging anyway. Later on, we have the original git commit reference if you use `git cherry-pick -x SHA`.

**Comment by Lennonka** [comment]
Cherry-picked:
- e777e53e9..64b93ecb3  3.5 -> 3.5
- e4fdb6472..b691d843c  3.4 -> 3.4
- 7b0d56cda..8c81b6499  3.3 -> 3.3
- d6376a98f..38c5b8ddb  3.2 -> 3.2
- d16e3e31c..0e120c7a8  3.1 -> 3.1

**Pull request merged**